### PR TITLE
Nav Bar

### LINKED ISSUE #7: User Story 2, Visitor Navigation
```
As a visitor
I see a navigation bar
This navigation bar includes links for the following:
- a link to return to the welcome / home page of the application ("/")
- a link to browse all books ("/books")
```

### Checklist:

- [x] base code tests are written
- [x] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [x] code is reviewed
- [x] branch is merged
- [x] all tests still pass

child of #31

child of #33

### PR DESCRIPTION
**Pull request by alexander-mathieu**
This PR adds a navigation bar to all pages except the welcome page.  Nav bar includes:

- "Home" - root_path
- "Browse by Authors" - authors_path
- "Browse by Books" - books_path

A partial _nav.html.erb has been added to all resource folders to accomplish this.  The application HTML calls upon the partial unless the IVAR disabled_nav is set to true, which is only in the welcome#index action.

Closes #7

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,23 @@
...
+  display: block;
+  color: #000;
+  padding: 8px 16px;
+  text-decoration: none;
+}
+
+#nav.li a:hover {
+  background-color: #555;
+  color: white;
+} */
```
Comment (BrennanAyers) [inline app/assets/stylesheets/application.css:35]: Let's get this pushed up to Heroku and see how it works

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class WelcomeController < ApplicationController
   def index
+    @disabled_nav = true
```
Comment (BrennanAyers) [inline app/controllers/welcome_controller.rb]: I think we can do this with `render layout false`

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -9,7 +9,10 @@
   </head>
 
   <body>
+    <%= render partial: "nav" unless @disabled_nav %>
```
Comment (BrennanAyers) [inline app/views/layouts/application.html.erb]: Let's move the partial into this application.html to keep it DRY

File: spec/features/welcome/index_spec.rb [hunk 4]
```diff
@@ -1,22 +1,27 @@
...
+  end
+
   describe 'I see two links to Authors and Books' do
     it 'When I click on Authors I go to the Authors index' do
       visit root_path
 
       click_link "Authors"
 
-      expect(current_path).to eq("/authors")
+      expect(current_path).to eq(authors_path)
```
Comment (BrennanAyers) [inline spec/features/welcome/index_spec.rb:16]: I am smart
